dids/src/main/kotlin/web5/sdk/dids/methods/web/DidWeb.kt

```diff
@@ -142,6 +143,6 @@ public sealed class DidWebApi(
   }
 
   public override fun create(keyManager: KeyManager, options: CreateDidOptions?): DidWeb {
-    throw RuntimeException("create operation not supported for did:web")
+    throw NotImplementedException("create operation not supported for did:web")
```

Comment:
not sure what's a better exception to throw here.

Reply:
i think `NotImplementedException` makes a lot of sense!

Reply:
changed to unsupportedoperationexception :)